### DISCUSSION TITLE
Certain Account not showing Related Events

### DISCUSSION THREAD

**Discussion by aronmule** [deejay-hub/timeline-lwc#212] [Q&A] [answered]
Hi, in our org we are seeing an issue where a certain Account does not show the related Events.
I assume the setup is ok if it works in some situations.
Two Event records I tested with, one being an example not shown on the Account and another that is shown in a different one, a StartDateTime, WhatId/Related To and a Subject are popluated.
The Account not showing any Events has many related when I query, 300+.
We are using an old version, 1.5.0.  Based on our sprint cycle, we will not update for a few weeks.  We want to make sure the issue is not something else.
Any idea on how to troubleshoot?  Let me know if I should provide screen shots.

Thanks!

Aron

**Comment by deejay-hub** [comment, marked as answer]
Hi @aronmule v1.5.0 is certainly a while ago. What you're likely experiencing is this issue - `[https://github.com/deejay-hub/timeline-lwc/issues/135](Issue 135)`. See issue for full description.

It was resolved by changing the Fallback Tooltip Field from "description" for Event and Task to any non long text area field - by default in v1.5 we used Description which is a Long Text Area which causes the issue you describe. So start there but ideally you'll want to upgrade and test in a sandbox as well.

Let us know how you go.

**Reply by aronmule** [reply]
Thanks so much for replying @deejay-hub .  This is the issue.  I will update the metadata.  It does not seem upgrading the release automatically does it but I will upgrade as well.  Thanks again!

**Comment by deejay-hub** [comment]
Glad to hear it solved your issue. Yes sadly the upgrade won't overwrite your metadata (by design) so it's something you'll have to do regardless.
